test/index.spec.js

```diff
@@ -2493,6 +2493,28 @@ describe('unexpected-dom', () => {
       `
       );
     });
+
+    // Regression test for https://github.com/unexpectedjs/unexpected-dom/issues/294
+    it('should produce a nice diff when satisfying a test node against an element with children', () => {
+      const document = new jsdom.JSDOM('<div>foo</div>').window.document;
```

Comment:
This probably needs to use either parseHtmlDocument or parseHtml to make sure it runs in the browser :)

Reply:
Yeah, I found out the hard way, fixed in 8609f1a5309a1b749fd336f03b15f0d7de05d757 :)

Reply:
@papandreou it might still be nice to use one of the wrapper functions as they should be making sure the right thing happens in both node and browser envs :)